int32 order
int64 traj_id
float64[] knots
geometry_msgs/Point[] pts
time start_time

